scripts/network/init.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -eo pipefail
+
+IP_BASE="223.42.0.1/16"
+
+/scripts/prepare-tap.sh -a "$IP_BASE" -o
+
+berserker /etc/berserker/network-server.toml &
+
+SERVER_PID=$!
+
+berserker /etc/berserker/network-client.toml &
```

Comment:
Would be good to make the workload configurable also.

Reply:
The workload parameters are configurable via the environment, so I think we can keep the existing files as "defaults" for the deployment and if adjustments are needed they can be done via the env

Reply:
I have been able to override these config files with a ConfigMap. I have also been able to override them with environment variables as you suggested. So, this is fine.